packages/spark-core/utilities/_utilities.scss

```diff
@@ -2,166 +2,301 @@
+  display: none !important;
 }
 
-.sprk-u-Block {
-  display: block;
+.sprk-u-Display--block {
+  display: block !important;
 }
 
-//
-// Accessible element hiding
-///
-.sprk-u-VisuallyHidden {
-  position: absolute;
-  left: -10000px;
-  top: auto;
-  width: 1px;
-  height: 1px;
-  overflow: hidden;
+.sprk-u-Display--inline {
+  display: inline !important;
 }
 
-// TODO: refactor maybe? look at type
-//
-// Text color utilities
-///
-.sprk-u-TextColor--green-light {
-  color: $green-light;
+.sprk-u-Display--inline-block {
+  display: inline-block !important;
 }
 
-.sprk-u-TextColor--green-medium {
-  color: $green-medium;
+.sprk-u-Visibility--hidden {
+  visibility: hidden !important;
 }
 
-.sprk-u-TextColor--green-dark {
-  color: $green-dark;
+.sprk-u-Visibility--visible {
+  visibility: visible !important;
 }
 
-.sprk-u-TextColor--blue-light {
-  color: $blue-light;
+// Floats, positioning, and alignment
+.sprk-u-Float--none {
+  float: none !important;
 }
 
-.sprk-u-TextColor--blue-medium {
-  color: $blue-medium;
+.sprk-u-Float--left {
+  float: left !important;
 }
 
-.sprk-u-TextColor--blue-dark {
-  color: $blue-dark;
+.sprk-u-Float--right {
+  float: right !important;
 }
 
-.sprk-u-TextColor--purple {
-  color: $purple-medium;
+.sprk-u-Position--relative {
+  position: relative !important;
 }
 
-.sprk-u-TextColor--red-light {
-  color: $red-light;
+.sprk-u-Position--absolute {
+  position: absolute !important;
 }
 
-.sprk-u-TextColor--red-medium {
-  color: $red-medium;
+.sprk-u-Position--fixed {
+  position: fixed !important;
 }
 
-.sprk-u-TextColor--red-dark {
-  color: $red-dark;
+.sprk-u-Position--static {
+  position: static !important;
 }
 
-.sprk-u-TextColor--orange-medium {
-  color: $orange-medium;
+.sprk-u-Top--zero {
+  top: 0 !important;
 }
 
-.sprk-u-TextColor--orange-dark {
-  color: $orange-dark;
+.sprk-u-Bottom--zero {
+  bottom: 0 !important;
 }
 
-.sprk-u-TextColor--gray-light {
-  color: $gray-light;
+.sprk-u-Left--zero {
+  left: 0 !important;
 }
 
-.sprk-u-TextColor--gray-medium {
-  color: $gray-medium;
+.sprk-u-Right--zero {
+  right: 0 !important;
 }
 
-.sprk-u-TextColor--gray-dark {
-  color: $gray-dark;
+.sprk-u-TextAlign--left {
+  text-align: left !important;
 }
 
-.sprk-u-TextColor--black {
-  color: $black;
+.sprk-u-TextAlign--right {
+  text-align: right !important;
 }
 
-.sprk-u-TextColor--white {
-  color: $white;
+.sprk-u-TextAlign--center {
+  text-align: center !important;
 }
 
-//
-// Background color utilities
-///
+.sprk-u-VerticalAlign--top {
+  vertical-align: top !important;
+}
+
+.sprk-u-VerticalAlign--middle {
+  vertical-align: middle !important;
+}
+
+.sprk-u-VerticalAlign--bottom {
+  vertical-align: bottom !important;
+}
+
+.sprk-u-VerticalAlign--baseline {
+  vertical-align: baseline !important;
+}
+
+// Text transform
+.sprk-u-TextTransform--uppercase {
+  text-transform: uppercase !important;
+}
+
+.sprk-u-TextTransform--none {
+  text-transform: none !important;
+}
+
+// Text color
```

Comment:
Not sure if we should have these color utilities at all. Text or background color. Background was that the highlight board component needs the ability to have different colored backgrounds and possibly different text colors based on the background color or background image. So I just went all in and created a class for everything.

Opinions? Maybe leave it out until it actually comes up again and we can figure out if it's the best way then?

Reply:
I'd rather see people using override classes in their app than using sprk-u classes for this. The potential for abuse with colors is too high, IMHO.